src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -78,10 +78,11 @@
         programGroup = Metrics.class
 )
 public class CollectAlignmentSummaryMetrics extends SinglePassSamProgram {
-    static final String USAGE_SUMMARY = "Produce a summary of alignment metrics from a SAM or BAM file";
-    static final String USAGE_DETAILS = "Using read outputs from high throughput sequencing (HTS) technologies, this tool provides " +
-            "metrics regarding the quality of read alignments to a reference sequence, as well as the proportion of the reads " +
-            "that passed machine signal-to-noise threshold quality filters (Illumina)."+
+    static final String USAGE_SUMMARY = "Produces a summary of alignment metrics from a SAM or BAM file";
+    static final String USAGE_DETAILS = "This tool takes an aligned SAM/BAM file, and produces metrics detailing the quality of the read alignments" +
+            " as well as the machine-generated base quality scores within these reads.  Additional information about Illumina's base quality " +
+            "scoring and the filters applied to these scores can be found at the following " +
+            "<a href=\"http://gatkforums.broadinstitute.org/gatk/discussion/6329/pf-reads-illumina-chastity-filter\"<b>link</b></a>."+
```

Comment:
Still working on this.  Not ready for review.  Got stuck on the conversion rates. Do you know how it is calculated with respect to the internal control e.g. Phage DNA?

Sent from my iPhone

> On Mar 11, 2016, at 4:56 PM, Geraldine Van der Auwera <notifications@github.com> wrote:
> 
> In src/java/picard/analysis/CollectAlignmentSummaryMetrics.java:
> 
> > @@ -78,10 +78,11 @@
> >          programGroup = Metrics.class
> >  )
> >  public class CollectAlignmentSummaryMetrics extends SinglePassSamProgram {
> > -    static final String USAGE_SUMMARY = "Produce a summary of alignment metrics from a SAM or BAM file";
> > -    static final String USAGE_DETAILS = "Using read outputs from high throughput sequencing (HTS) technologies, this tool provides " +
> > -            "metrics regarding the quality of read alignments to a reference sequence, as well as the proportion of the reads " +
> > -            "that passed machine signal-to-noise threshold quality filters (Illumina)."+
> > +    static final String USAGE_SUMMARY = "Produces a summary of alignment metrics from a SAM or BAM file";
> > +    static final String USAGE_DETAILS = "This tool takes an aligned SAM/BAM file, and produces metrics detailing the quality of the read alignments" +
> > +            " as well as the machine-generated base quality scores within these reads.  Additional information about Illumina's base quality " +
> > +            "scoring and the filters applied to these scores can be found at the following " +
> > +            "<a href=\"http://gatkforums.broadinstitute.org/gatk/discussion/6329/pf-reads-illumina-chastity-filter\"<b>link</b></a>."+
> This should point to the website reader view, not the forum view
> 
> —
> Reply to this email directly or view it on GitHub.
>